pkg/interfaces/contracts/vault/IVaultErrors.sol

```diff
@@ -215,6 +215,9 @@ interface IVaultErrors {
      */
     error FeePrecisionTooHigh();
 
+    /// @dev Given percentage is above FixedPoint.ONE (1e18 wei).
+    error InvalidPercentage();
```

Comment:
Should it be renamed to something that explains that the percentage was greater than 100%? Like, PercentageAboveMax, or something like this?

It's minor, the comment explains, but just a suggestion, since this error is only used when the percentage is bigger than a maximum number.

Reply:
Yes; currently it's always 100% (hence "Invalid" meaning outside 0-100%). However, in the code there is a check against `MAX_FEE_PERCENTAGE`, which could possibly be set < 100% at some point (e.g., for dynamic fees). So it makes sense to change.